randovania/games/factorio/game_data.py

```diff
@@ -74,12 +73,31 @@ def _hash_words() -> list[str]:
     presets=[
         {"path": "starter_preset.rdvpreset"},
     ],
-    faq=[],
+    faq=[
+        ("What versions of the game is supported?", "Only Factorio 2.0 is supported."),
+        ("Is Factorio: Space Age supported?", "No."),
+        ("Are other mods supported?", "Only mods that don't add new researches are compatible."),
+        (
+            "Is multiplayer supported?",
```

Comment:
maybe worth it to clarify "built-in multiplayer"? just to avoid confusion between factorio multiplayer and mw/coop